sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -248,47 +248,46 @@ export class ProgramInfoRequestController extends BaseController {
         throw new BadRequestException("Application not found.");
       }
 
-      let selectedOfferingIntensity = payload.offeringIntensity;
-
-      if (payload.selectedOffering) {
-        const offering = await this.offeringService.getOfferingById(
-          payload.selectedOffering,
-        );
-        selectedOfferingIntensity = offering.offeringIntensity;
-      }
-
       this.applicationService.checkOfferingIntensityMismatch(
         application.data.howWillYouBeAttendingTheProgram,
-        selectedOfferingIntensity,
+        payload.offeringIntensity,
```

Comment:
I think you need this offeringIntensity logic
![image](https://user-images.githubusercontent.com/77353155/163455109-be303864-e015-4329-a6a1-0a953498e058.png)

Reply:
get offeringIntensity from offering object `offeringLocation` line 262. and move this function after line 280